Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/fragment/SubmissionDispatcherFragment.kt

```diff
@@ -52,7 +52,7 @@ class SubmissionDispatcherFragment : Fragment(R.layout.fragment_submission_dispa
 
     override fun onResume() {
         super.onResume()
-        binding.submissionDispatcherRoot.sendAccessibilityEvent(AccessibilityEvent.TYPE_ANNOUNCEMENT)
+        binding.submissionDispatcherRoot.sendAccessibilityEvent(AccessibilityEvent.TYPE_VIEW_FOCUSED)
```

Comment:
Why is this change necessary?

Reply:
During testing I found out that the component wasn't focused by default when using `TYPE_ANNOUCEMENT` and therefore nothing was announced by the screen reader. Also settings android:focusable="true"  alone didn't work out. Explicitly setting `TYPE_ANNOUCEMENT` to the view fixes the issue.